core/lib/spree/money.rb

```diff
@@ -81,12 +81,12 @@ def format(options = {})
     # @param options [Hash] additional formatting options
     # @return [String] the value of this money object formatted according to
     #   its options and any additional options, by default as html.
-    def to_html(options = { html: true })
```

Comment:
can you please also:

- update documentation on line 80, that mention the `html: true` param
- deprecate the `html` option into solidus, so that if stores devs are still using it, they will be notified that they need to change their code to pass `html_wrap` instead